backend/data/src/main/resources/I18n_en.properties

```diff
@@ -80,4 +80,26 @@ slack.common.message.global-subscription-error=You can't set global subscription
 slack.common.message.modification-info=*%s* has changed the %s at %s
 slack.common.message.feature-not-available=This feature is not available for your plan. Please upgrade your plan to use this feature.
 slack.common.message.bot-not-in-channel=Bot is not in the channel. Please invite the bot to the channel using `/invite @Tolgee` and try again.
-slack.help.message.advanced-subscribe-events-example=`/tolgee subscribe 1 fr --on new_key` - subscribe to new key events in project 1 for French translations
\ No newline at end of file
+slack.help.message.advanced-subscribe-events-example=`/tolgee subscribe 1 fr --on new_key` - subscribe to new key events in project 1 for French translations
+
+notifications.email.subject.TASK_ASSIGNED=New task assignment
+notifications.email.subject.TASK_COMPLETED=Task has been completed
+notifications.email.subject.TASK_CLOSED=Task has been closed
+notifications.email.subject.MFA_ENABLED=Multi-factor authentication has been enabled for your account
+notifications.email.subject.MFA_DISABLED=Multi-factor authentication has been enabled for your account
+notifications.email.subject.PASSWORD_CHANGED=Password has been changed for your account
+
+notifications.email.task-type.TRANSLATE=translate
+notifications.email.task-type.REVIEW=review
+
+notifications.email.task-header.TASK_ASSIGNED=Password has been changed for your account
+notifications.email.task-header.TASK_COMPLETED=Task you've created has been completed
+notifications.email.task-header.TASK_CLOSED=Task you've created has been closed
+
+notifications.email.my-tasks-link=Check all your tasks <a href="${myTasksUrl}">here</a>.
```

Comment:
```suggestion
notifications.email.my-tasks-link=Check your tasks <a href="${myTasksUrl}">here</a>.
```

Reply:
Thanks.

Just for context - I was just moving the logic, keeping the text as they were before.